File: cakk-domain/src/main/java/com/cakk/domain/config/P6spyConfig.java
package com.cakk.domain.config;

import jakarta.annotation.PostConstruct;

import org.springframework.context.annotation.Configuration;

import com.p6spy.engine.spy.P6SpyOptions;

@Configuration
public class P6spyConfig {

	@PostConstruct
	public void setLogMessageFormat() {
		P6SpyOptions.getActiveInstance().setLogMessageFormat(P6spySqlFormatterConfig.class.getName());
	}
}

File: cakk-domain/mysql/src/main/java/com/cakk/domain/mysql/config/P6spyConfig.kt
package com.cakk.domain.mysql.config

import com.p6spy.engine.spy.P6SpyOptions
import jakarta.annotation.PostConstruct
import org.springframework.context.annotation.Configuration

@Configuration
class P6spyConfig {
    @PostConstruct
    fun setLogMessageFormat() {
        P6SpyOptions.getActiveInstance().logMessageFormat = P6spySqlFormatterConfig::class.java.name
    }
}
